### DISCUSSION TITLE
check if service running 503

### DISCUSSION THREAD

**Discussion by deenodj** [mynaparrot/plugNmeet-server#620] [General]
i have a question, when i try to access plugnmeet.domain.nl/auth than i get a error 503 service Unavailable No Server is available to handle this request.  How can i check the api service? whe i type: systemctl start plugnmeet, i thinks service getting start there is futher no response, than i ask service and think everything is running... ? What can i do to check this?

![api](https://github.com/user-attachments/assets/b308c588-0a47-46f9-86b5-e4446ca07baa)

**Comment by jibon57** [comment]
You'll need to check the server's log to identify why it's down. You can check either from docker or `/opt/plugNmeet/logs` dir or `systemctl status plugnmeet`

**Comment by deenodj** [comment]
Yes, he said:

 {
  "file": "app/main.go:54",
  "func": "main.startServer",
  "level": "fatal",
  "msg": "Error 1045 (28000): Access denied for user 'plugnmeet'@'localhost' (using password: YES)",
  "time": "2024-12-21T21:27:14Z"
}

btw i can't find my files localy on windows machine, i running de app Ubuntu with PlugNmeet.

**Comment by deenodj** [comment]
I asked google but there are several options possible. Also because maria DB is involved here. Do we change the rights of plugNmeet@local here? Hope someone can help me. thank you very much
